pod/annotations.go

```diff
@@ -206,6 +206,15 @@ const (
 	AnnotationKeyPredRuntimeQuantiles    = "runtime.predictions.titus.netflix.com/quantiles"
 	AnnotationKeyPredRuntimeModelVersion = "runtime.predictions.titus.netflix.com/model-version"
 	AnnotationKeyPredRuntimeModelID      = "runtime.predictions.titus.netflix.com/model-id"
+
+	// mockVK
+	AnnotationKeyTitusParameterMockVkPrepareTime = "titusParameter.mockVK.prepareTime"
+	AnnotationKeyTitusParameterMockVkRunTime     = "titusParameter.mockVK.runTime"
+	AnnotationKeyTitusParameterMockVkKillTime    = "titusParameter.mockVK.killTime"
```

Comment:
These are titus parameters.
I appreciate putting the constants here, but they don't need to go here.